htdocs/themes/xbootstrap/tpl/leftBlock.tpl

```diff
@@ -1,10 +1,12 @@
 <{if $xoBlocks.canvas_left}>
     <div class="col-sm-3 col-md-3 xoops-side-blocks">
         <{foreach item=block from=$xoBlocks.canvas_left}>
-            <aside>
+        <div class="xoops-blocks">
```

Comment:
This appears to open new div elements in the foreach loop that will never be closed?